website/server/extractDocs.js

```diff
@@ -505,12 +505,15 @@ const styleDocs = docsList.stylesForEmbed.reduce(function(docs, filepath) {
 
 function extractDocs() {
   componentCount = 0;
+  var components = docsList.components.map(renderComponent);
```

Comment:
I split these out while I was looking into the timing of each of these steps, as it allowed me to use `console.time()` and `console.timeEnd()` before/after each of these to identify the bottlenecks in extractDocs. I missed undoing this change after I removed the timing calls.

Splitting these out is not required for the main goal of this PR, if you feel this change should be reverted let me know.